Makefile

```diff
@@ -386,3 +395,23 @@ verify-generated: bundle generate
 
 install-git-hooks:
 	git config core.hooksPath .githooks	
+
+
+GOLANGCI_LINT_LOCAL_VERSION := $(shell command ${GOLANGCI_LINT} --version 2> /dev/null | awk '{print $$4}')
+.PHONY: golangci-lint
+golangci-lint:
+	@if [ ! -x "$(GOLANGCI_LINT)" ]; then\
+		echo "Downloading golangci-lint $(GOLANGCI_LINT_VERSION)";\
```

Comment:
this seems to be redundant given the line right below

Reply:
deleted.
Thanks for the comment